packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -43,7 +44,8 @@ function preload(baseModule: () => Promise<{}>, deps?: string[]) {
 
   return Promise.all(
     deps.map((dep) => {
-      dep = `__VITE_PRELOAD_BASE__${dep}`
+      // @ts-ignore
+      dep = `${base}${dep}`
       // @ts-ignore
       if (dep in seen) return
```

Comment:
I may be wrong?! But could it be better to move these two lines into a `.map` and `.filter` call?
That way it may be more readable and understandable whats going on :thinking: